charts/telemeter-reporting/templates/report-queries/machine-cpu-cores-by-cluster-id.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "machine-cpu-cores-by-cluster-id-raw"
+  labels:
+    telemeter-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportDataSources:
+  - "machine-cpu-cores-by-cluster-id"
+  columns:
+  - name: machine_cpu_cores
+    type: double
+    unit: cpu_cores
+  - name: machine_cpu_core_seconds
+    type: double
+    unit: cpu_core_seconds
+  - name: cluster_uuid
+    type: string
+    unit: uuid
+  - name: timestamp
+    type: timestamp
+    unit: date
+  - name: timeprecision
+    type: double
+    unit: seconds
+  - name: dt
+    type: string
+  query: |
+      SELECT amount as machine_cpu_cores,
+          amount * timeprecision as machine_cpu_core_seconds,
+          element_at(labels, '_id') as cluster_uuid,
+          timestamp,
+          timeprecision,
+          dt
+      FROM {| dataSourceTableName "machine-cpu-cores-by-cluster-id" |}
+
+---
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportGenerationQuery
+metadata:
+  name: "machine-cpu-cores-by-cluster-id"
+  labels:
+    operator-metering: "true"
+{{- block "extraMetadata" . }}
+{{- end }}
+spec:
+  reportQueries:
+  - "machine-cpu-cores-by-cluster-id-raw"
+  view:
+    disabled: true
+  columns:
+  - name: period_start
+    type: timestamp
+    unit: date
+  - name: period_end
+    type: timestamp
+    unit: date
+  - name: cluster_uuid
+    type: string
+    unit: uuid
+  - name: data_start
+    type: timestamp
+    unit: date
+  - name: data_end
+    type: timestamp
+    unit: date
+  - name: machine_cpu_core_seconds
+    type: double
+    unit: cpu_core_seconds
+  inputs:
+  - name: ReportingStart
+  - name: ReportingEnd
+  query: |
+    SELECT
+      timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart| prestoTimestamp |}' AS period_start,
+      timestamp '{| default .Report.ReportingEnd .Report.Inputs.ReportingEnd | prestoTimestamp |}' AS period_end,
+      cluster_uuid,
+      min("timestamp") as data_start,
```

Comment:
I don't think we need these `data_` columns

Reply:
I based this query off of a query in the openshift-metering chart. If we don't want data_start/data_end here, do we want them in the other charts? They can be ~vaguely useful for knowing if we have gaps in our data. Since this is per-cluster, it would be helpful if we wanted to know if telemeter was offline for a period, for instance.

Reply:
This doesn't actually tell you if you have gaps, thats why this column is bad. I'd like to remove it from existing queries, but that would need to be done more as a breaking change that we handle with versioning, which i don't want to deal with now. I think it's best we avoid adding it to new queries when possible though.